modules/shared_state.py

```diff
@@ -7,6 +7,8 @@
 from typing import Optional
 
 log = logging.getLogger(__name__)
+if shared.cmd_opts.allow_interrupt_generation_in_command_line:
+    log.setLevel(logging.INFO)

```

Comment:
don't do this
this is bad
ie I set it log level to debug and hear you force it to INFO
this is not the way to solve it